pingcap/tiup
# cluster: clean operations cannot cleanup monitor component files

### What problem does this PR solve? 
clean operations cannot cleanup monitor data/log/tls files

for example：
![image](https://user-images.githubusercontent.com/39378935/143839956-09176495-799f-4f6a-b378-e4648dbc47d3.png)

### What is changed and how it works?
![image](https://user-images.githubusercontent.com/39378935/143840132-abf1ce9c-2bae-42e8-9605-4ebd64bc7c26.png)



### Check List 

Tests 


 - Manual test (add detailed scripts or steps below)
 
Check whether the output file contains the monitor component
```shell
 ./tiup-cluster clean <cluster-name> --ignore-node <host> --ignore-role <role> --data
 ./tiup-cluster clean <cluster-name> --ignore-node <host> --ignore-role <role> --log
 ./tiup-cluster clean <cluster-name> --ignore-node <host> --ignore-role <role> --all
```
 
Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change


Side effects

 - Increased code complexity


Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation


Release notes:

```release-note
NONE
```
